src/css/style.scss

```diff
@@ -106,10 +106,29 @@ input[type=button], input[type=submit], input[type=reset] {
   font-size: 10px;
 }
 
+input[type=button]:disabled, input[type=submit]:disabled {
+  background-color: #4e4e4e;
+}
+
 #success-message {
   display: none;
 }
 
 #error-message {
   display: none;
 }
+
+.loader {
+  display: none;
+  border: 4px solid white;
+  border-top: 4px solid black;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
```

Comment:
Please, configure your editor the leave a newline at the end of files.
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206